skymap_scanner/client/reco_icetray.py

```diff
@@ -290,22 +291,52 @@ def main() -> None:
     with open(args.in_pkl, "rb") as f:
         msg = pickle.load(f)
         reco_algo = msg[cfg.MSG_KEY_RECO_ALGO]
-        pframe = msg[cfg.MSG_KEY_PFRAME]
+        pframes = msg[cfg.MSG_KEY_PFRAME]
 
     # get GCDQp_packet
     with open(args.GCDQp_packet_json, "r") as f:
         GCDQp_packet = full_event_followup.i3live_json_to_frame_packet(
             f.read(), pnf_framing=False
         )
 
-    # go!
-    reco_pixel(
-        reco_algo,
-        pframe,
-        GCDQp_packet,
-        str(args.baseline_GCD_file),
-        args.out_pkl,
-    )
+    reco_obj = recos.get_reco_interface_object(reco_algo)
+    processes = []
+    outpaths = []
+
+    # loop over the pframes to run in parallel
+    for i, pframe in enumerate(pframes):
+        outpath = Path(f"out{i:04}.pkl")
```

Comment:
Could this use a more unique path, either using `tempfile` or maybe with the parent process pid?  I'm thinking of the possibility of running two instances side by side, which could happen in EWMS.